test/drivers/test_suites/syscall_exit_suite/dup3_x.cpp

```diff
@@ -17,8 +17,8 @@ TEST(SyscallExit, dup3X)
 	int32_t new_fd = old_fd;
 	uint32_t flags = O_CLOEXEC;
 	int32_t res = syscall(__NR_dup3, old_fd, new_fd, flags);
-	assert_syscall_state(SYSCALL_FAILURE, "dup3", res);
 	int64_t errno_value = -errno;
+	assert_syscall_state(SYSCALL_FAILURE, "dup3", res);
```

Comment:
Since `assert_syscall_state` now resets `errno` to 0, we must take `errno` value before calling the macro.
Same below.